models/cartIntegration/customerPricing.go

```diff
@@ -35,12 +35,14 @@ var (
 		LEFT JOIN CartIntegration ci ON ci.partID = p.partID AND ci.custID = ?
 		left join Price pr on pr.partID = p.partID and pr.priceType = 'list'
 		WHERE p.status in (700, 800, 810, 815, 850, 870, 888, 900, 910, 950)  && p.brandID = ?
+		GROUP BY p.partID
```

Comment:
If someone has multipe p.partIds we need to clean up the data, I don't get the user case where we should allow this.